lib/matplotlib/axes/_axes.py

```diff
@@ -7643,6 +7643,7 @@ def specgram(self, x, NFFT=None, Fs=None, Fc=None, detrend=None,
         if mode == 'complex':
             raise ValueError('Cannot plot a complex specgram')
 
+        _api.check_in_list([None, 'default', 'linear', 'dB'], scale=scale)
```

Comment:
I'm not sure how this is fixing the error?

Reply:
Oops, that was not supposed to be part of this PR.

Reply:
Hah, I committed the opposite of what I wanted here...